### PR TITLE
Monitoring OpenStack in MMaaS

### PR DESCRIPTION
**Pull request by suehighmoor**
New article explaining how to set up a VPN so that OpenStack instances can be monitored with MMaaS

### CODE REVIEW

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+2. [Set up the network](#setting-up-the-network).
+
+3. [Attach the network to the router](#attaching-the-network-to-the-router).
+
+4. [Apply security groups](#applying-security-groups).
+
+5. [Deploy the VPN instance](#deploying-the-vpn-instance).
+
+6. [Set up the VPN endpoints](#setting-up-the-vpn-endpoints).
```
Comment (ARapisardaUKC) [inline articles/managed-operations/man-how-configure-vpn-for-openstack.md]: Configure the VPN endpoint

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+     - **Network Address:** The network subnet in a CIDR format. The subnet should not already be in use and the subnet mask should be `x.x.x.0/28`.
+
+     - **IP Version:** Select `IPv4`.
+
+     - **Gateway IP:** The IP dedicated to the gateway of the network. For ease of use, use the network address but ensure that the last octet is a `.1`, for example, `192.168.0.1`.
+
+   - **Subnet Details** tab:
+
+     - **DNS Name Servers:** Enter `8.8.8.8`.
```
Comment (ARapisardaUKC) [inline articles/managed-operations/man-how-configure-vpn-for-openstack.md:90]: Or your preferred public DNS server

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+1. Expand **Network** then select **Network Topology**.
+
+2. Hover over the internet router and click **Add Interface**.
+
+3. In the *Add Interface** dialog box, provide the following information:
+
+   - **Subnet:** Select the network the VPN will sit on.
+
+   - **IP Address:** Leave this blank so the gateway address you picked in the previous will be chosen.
```
Comment (ARapisardaUKC) [inline articles/managed-operations/man-how-configure-vpn-for-openstack.md:106]: previous step*

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+   - Click the port that corresponds to the instance's internal IP address.
+
+   - Click **Allowed Address Pairs** then **+**.
+
+   - Create an address pair for the local subnet and click **Submit**.
+
+   - Create a second address pair for the peer subnet and click **Submit**.
+
+7. You need to add a static route to the OpenStack router to ensure that traffic for the Opsview collectors are routed via the VPN.
```
Comment (ARapisardaUKC) [inline articles/managed-operations/man-how-configure-vpn-for-openstack.md]: Dont mention Opsview collectors - please change this to "UKCloud Monitoring infrastructure is routed via the VPN"

**Review by ARapisardaUKC — COMMENTED** [review]
I have made a few notes

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (suehighmoor) [reply articles/managed-operations/man-how-configure-vpn-for-openstack.md]: Fixed

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md (hunk 2 above)
Comment (suehighmoor) [reply articles/managed-operations/man-how-configure-vpn-for-openstack.md]: Fixed

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md (hunk 3 above)
Comment (suehighmoor) [reply articles/managed-operations/man-how-configure-vpn-for-openstack.md]: Fixed

File: articles/managed-operations/man-how-configure-vpn-for-openstack.md (hunk 4 above)
Comment (suehighmoor) [reply articles/managed-operations/man-how-configure-vpn-for-openstack.md]: Fixed

**Comment by suehighmoor** [comment]
@ARapisardaUKC - thanks for the review. I've updated the article based on your comments.

**Review by ARapisardaUKC — APPROVED** [review]

**Comment by ARapisardaUKC** [comment]
LGTM

**Review by stevedicko — APPROVED** [review]
Can't see any obvious issues - fair play to Adri for the hard work!

**Pull request merged**